setup.cfg

```diff
@@ -1,8 +1,18 @@
 [flake8]
-ignore=E133,E226,E228,N802,N803,N806
-exclude = __init__.py
+ignore=E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+exclude = 
+    bin,
```

Comment:
Please remove bin. This is a holdover from DM what tests bin.src but not bin. T&S tends to not bother with a bin.src.